README.rst

```diff
@@ -324,6 +324,18 @@ Additional metadata to be stripped can be configured via either
 
         --extra-keys "metadata.celltoolbar cell.metadata.heading_collapsed"
 
+You can keep certain metadata with either
+
+*   ``git config (--global/--system) filter.nbstripout.keepmetadatakeys``, e.g. ::
```

Comment:
Lmk if you want something other than `keepmetadatakeys`.

Reply:
No, I think this is fine :)